src/components/common/Input/Input.tsx

```diff
@@ -1,5 +1,6 @@
-import React from 'react';
-import { InputGroup } from './InputStyle';
+import React, { useState } from 'react';
+import { UseFormRegisterReturn } from 'react-hook-form';
+import { FileName, InputGroup } from './InputStyle';
 
 interface IInput {
```

Comment:
no need to mention all attributes here, extend from the default interface

Reply:
https://github.com/KLIM-nk01/virtual-ambulance/pull/27/commits/35d1051dd92f9591600c90270a824d4eec0360e7#diff-e2eec15dd4a882dc8cd779ef821494357bd0dface0e4d8e40fd8d7c4f8bbe31a